src/index.html

```diff
@@ -9,10 +12,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
Avoid using modifiers like `calendar--start-day-sun` and `calendar--month-length-31` directly in the markup. These should be added dynamically with JavaScript based on the actual start day and month length.